src/node-github-release.sh

```diff
@@ -56,10 +56,21 @@ fi
 
 echo "Releasing v$PKG_VERSION"
 
-# Create a release.
-github-release.v0 release \
-  --user "$CIRCLE_PROJECT_USERNAME" \
-  --repo "$CIRCLE_PROJECT_REPONAME" \
-  --tag "v$PKG_VERSION" \
-  --name "Release $PKG_VERSION" \
-  --description "$(get_changelog)"
\ No newline at end of file
+if [ -z "$GITHUB_RELEASE_TARGET" ]; then
+  # Create a release.
+  github-release.v0 release \
+    --user "$CIRCLE_PROJECT_USERNAME" \
+    --repo "$CIRCLE_PROJECT_REPONAME" \
+    --tag "v$PKG_VERSION" \
+    --name "Release $PKG_VERSION" \
+    --description "$(get_changelog)"
+else 
+  # Create a release with a target branch
+  github-release.v0 release \
+    --user "$CIRCLE_PROJECT_USERNAME" \
+    --repo "$CIRCLE_PROJECT_REPONAME" \
+    --tag "v$PKG_VERSION" \
+    --target "$GITHUB_RELEASE_TARGET" \
+    --name "Release $PKG_VERSION" \
+    --description "$(get_changelog)"
+fi
```

Comment:
Rather than duplicating all of this, let's just build an array of arguments to pass to `github-release`, optionally including `--target`:

```suggestion
args=(
  --user "$CIRCLE_PROJECT_USERNAME" 
  --repo "$CIRCLE_PROJECT_REPONAME" 
  --tag "v$PKG_VERSION" 
  --name "Release $PKG_VERSION"
)

if [ -n "$GITHUB_RELEASE_TARGET" ]; then
  args+=("--target" "$GITHUB_RELEASE_TARGET")
fi

# Create a release.
github-release.v0 release "${args[@]}" --description "$(get_changelog)"
```